applications/MultilevelMonteCarloApplication/python_scripts/mlmc_utilities.py

```diff
@@ -1,43 +1,57 @@
-from __future__ import absolute_import, division # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
-
-# Importing the Kratos Library
-import KratosMultiphysics
-from KratosMultiphysics.MultilevelMonteCarloApplication.tools import ParametersWrapper
-
-# Import packages
+# Import Python libraries
```

Comment:
Is this file (`mlmc_utilities.py`) still in use with XMC? Just curious

Reply:
This is the original MC-MLMC synchronous and asynchronous algorithms. It is not used in XMC